src/olympia/versions/views.py

```diff
@@ -98,14 +99,24 @@ def is_appropriate_reviewer(addon, channel):
 
     attachment = bool(type == 'attachment')
     if use_cdn:
-        # When using the CDN URL, we do a redirect, so we can't set
-        # Content-Disposition: attachment for attachments. To work around this,
-        # if attachment=True, get_file_cdn_url() changes the path to something
-        # we recognize in the nginx config.
-        loc = urlparams(file_.get_file_cdn_url(attachment=attachment),
-                        filehash=file_.hash)
-        response = http.HttpResponseRedirect(loc)
-        response['X-Target-Digest'] = file_.hash
+        # When serving the file for the general public through the CDN, we need
+        # to obey regional restrictions
+        region_code = request.META.get('HTTP_X_COUNTRY_CODE', None)
+        if region_code and AddonRegionalRestrictions.objects.filter(
+                addon=addon,
+                excluded_regions__contains=region_code.upper()).exists():
```

Comment:
Shouldn't we use `icontains` here as well?

Reply:
https://github.com/mozilla/addons-server/pull/15965/files#r520608452